src/lib.rs

```diff
@@ -64,7 +64,7 @@ mod stdlib {
 
 use byteorder::{ByteOrder, LittleEndian};
 use stdlib::collections::HashMap;
-use stdlib::collections::Vec;
+use stdlib::vec::Vec;
```

Comment:
If such changes are necessary to “make it build successfully”, they should likely be fold into the previous commit. Again, more descriptive commit logs to understand what happens would be nice :)